backend/apps/progress/urls.py

```
@@ -48,12 +48,8 @@
     
     # Recommendations
     path("recommendations/", RecommendationsView.as_view(), name="recommendations"),
-
-    
-    # Community
-
-    path("daily-stats/", DailyLessonStatsView.as_view(), name="daily-stats"),
-
+    path('certificate/verify/<str:hash>/', CertificateVerifyView.as_view(), name='certificate-verify'),
```

Ruff (0.15.21)

[error] Undefined name `CertificateVerifyView`

(F821)